cookbook/direnv.md

```diff
@@ -18,8 +18,13 @@ $env.config = {
   hooks: {
     pre_prompt: [{ ||
       let direnv = (direnv export json | from json)
-      let direnv = if ($direnv | length) == 1 { $direnv } else { {} }
-      $direnv | load-env
+      let no_changes = $direnv | is-empty
+      let changes = not $no_changes
+
+      if $changes {
+        let direnv = $direnv | upsert PATH {|direnv_result| ($direnv_result.PATH | split row ":") }
```

Comment:
i would remove the parentheses here and use `it` to have a bit less clutter on the line
```suggestion
        let direnv = $direnv | upsert PATH {|it| $it | split row ":"}
```

Reply:
I agree it looks better, just need to modify `$it` to `$it.PATH`
```suggestion
        let direnv = $direnv | upsert PATH {|it| $it.PATH | split row ":" }
```